jsapp/js/stores.es6

```diff
@@ -399,7 +397,7 @@ var userExistsStore = Reflux.createStore({
   init () {
     this.checked = {};
     this.listenTo(actions.misc.checkUsername.completed, this.usernameExists);
-    this.listenTo(actions.misc.checkUsername.failed_, this.usernameDoesntExist);
+    this.listenTo(actions.misc.checkUsername.failed, this.usernameDoesntExist);
```

Comment:
This doesn't make a distinction between a network/server failure and bad user input, e.g. if I disconnect from the internet and try to grant permissions, I see `User not found: [username]` for valid usernames. Since we now display an alert (`notifyUnknownUser()`) instead of a more-ambiguous greyed-out button, it would be better to show `User not found` only when receiving a 404 from the backend. This could go into a new issue if you'd prefer.

Reply:
Would prefer new issue 😗

Reply:
OK <3 https://github.com/kobotoolbox/kpi/issues/2341